semisupervised/StackedAutoEncoderSSL.py

```diff
@@ -155,12 +155,12 @@ def fit(self, X, y, is_pretrain=True, validation_data=None):
                     min_loss = valid_loss
                 else:
                     patience_counter += 1
-                if self.verbose:
+                if ep % self.verbose == 0:
```

Comment:
Note that this approach gives a division by 0 error if verbose is set to 0, which is common to signal no output. If the modulo behaviour is to be retained I'd suggest this:

`if self.verbose and (ep % self.verbose == 0):`

Reply:
Yes, you are right. I prefer to `if self.verbose and (ep % self.verbose == 0):`.